### ISSUE TITLE
Somehow get 100% of multiplayer barriers removed

### ISSUE THREAD

**Issue by ItsZman2006** [ItsZman2006/Dark-Souls-III-Coop-Overhaul#4]

**Comment by TopazOwlz** [comment]
Hey? Are you still working on this mod? I'm very interested in getting it when it's complete.

**Comment by ItsZman2006** [comment]
Yep! Still working on it :) Working with a few people on my discord to
remake it from the ground up and add some more features to it :)
